docs/ruleset.rst

```diff
@@ -43,6 +43,14 @@ Left-hand sides are currently a few special forms which select nodes to be fed t
 
 .. autofunction:: and(typeCall[, typeCall, ...])
 
+.. function :: when(predicate)
+
+    Further specify types of node you'd like to select. Can be chained with type() or dom().
```

Comment:
Can you move this into a jsdoc comment in the code itself, like the other functions are?